modules/secure-cloud-serverless-net/iam.tf

```diff
@@ -25,13 +25,6 @@ resource "google_project_service_identity" "vpcaccess_identity_sa" {
   service = "vpcaccess.googleapis.com"
 }
 
-resource "google_project_service_identity" "run_identity_sa" {
-  provider = google-beta
-
-  project = var.serverless_project_id
-  service = "run.googleapis.com"
-}
-
```

Comment:
This would be the right way to create service identity SAs. Passing it as a variable might not be a good approach.